products-service/src/main/java/com/petclinic/products/utils/DataLoaderService.java

```diff
@@ -40,7 +41,8 @@ public void run(String... args) throws Exception {
                 .productDescription("Premium dry food for adult dogs")
                 .productSalePrice(45.99)
                 .requestCount(0)
-                .productType("Food")
+                .averageRating(1.3)
```

Comment:
Same goes here, I fixed these issues in [this PR](https://github.com/cgerard321/champlain_petclinic/pull/944) and this would essentially bring back the Rating bug.